python/lsst/meas/base/dataIds.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
```

Comment:
For what it's worth, I already wrestled with this problem when I started (but still haven't finished...) porting `meas_mosaic` without really finding a solution I was happy with.

If I were you, I would put it in `forcedPhotCcd.py`, since that's the only place it's currently used; if somewhere else would seem more appropriate for `meas_mosaic`, let's worry about that when we do the the `meas_mosaic` port.

That said, I won't strongly object if you decide to leave it in `dataIds.py`. I have no other suggestions as to the location.